### PR TITLE
Redis Lettuce span UI doesn't show detailed Redis command parameters in 'db.statement' field

### PR DESCRIPTION
**Pull request by compilerduck**
On Redis Lettuce span UI, only a Redis command keyword is displayed in 'db.statement' field（see below screenshot）, for example "GET", which is inconvenient for system monitoring. Suppose one sees tens of Redis spans in one trace, and he wants to know what are they. Unfortunately he can't get this currently in an accurate way. What he can get is a list of "SET" or "GET", but "SET" for what, "GET" for what, are unknown for him. So I create this pull request to enhance this, enhanced UI screenshots are shown in the last three pictures.

Current UI:
image

Enhanced UI:
image
image
image

- [ ] If this is non-trivial feature, paste the links/URLs to the design doc.
- [ ] Update the documentation to include this new feature.
- [x] Tests(including UT, IT, E2E) are added to verify the new feature.
- [x] If it's UI related, attach the screenshots below.

- [ ] If this pull request closes/resolves/fixes an existing issue, replace the issue number. Closes #<issue number>.
- [x] Update the [`CHANGES` log](https://github.com/apache/skywalking/blob/master/CHANGES.md).

### CODE REVIEW

File: apm-sniffer/apm-sdk-plugin/lettuce-5.x-plugin/src/main/java/org/apache/skywalking/apm/plugin/lettuce/v5/RedisChannelWriterInterceptor.java [hunk 1]
```diff
@@ -61,6 +71,20 @@ public void beforeMethod(EnhancedInstance objInst, Method method, Object[] allAr
         Tags.DB_STATEMENT.set(span, dbStatement.toString());
         SpanLayer.asCache(span);
     }
+    
+    private String getArgsStatement(RedisCommand redisCommand) {
```
Comment (wu-sheng) [inline apm-sniffer/apm-sdk-plugin/lettuce-5.x-plugin/src/main/java/org/apache/skywalking/apm/plugin/lettuce/v5/RedisChannelWriterInterceptor.java:75]: Take a look at `JDBCPluginConfig#TRACE_SQL_PARAMETERS` to do. Collecting parameters costs a lot of resources (CPU/memory).

Comment (wu-sheng) [reply apm-sniffer/apm-sdk-plugin/lettuce-5.x-plugin/src/main/java/org/apache/skywalking/apm/plugin/lettuce/v5/RedisChannelWriterInterceptor.java:75]: And you need to update the tests to verify the parameter collection here. https://github.com/apache/skywalking/tree/master/test/plugin/scenarios/redisson-scenario

**Comment by wu-sheng** [comment]
agent setup doc should be updated about the new config you added.
And you need to resolve conflict.

**Comment by wu-sheng** [comment]
@kezhenxu94 Please take a look.

**Review by kezhenxu94 — DISMISSED** [review]
Screen Shot 2021-04-01 at 14 36 41

This space looks weird, though it looks normal on the website

**Review by kezhenxu94 — APPROVED** [review]
LGTM

**Pull request merged**